plugins/uefi/fu-uefi-common.c

```diff
@@ -273,9 +273,9 @@ fu_uefi_read_file_as_uint64 (const gchar *path, const gchar *attr_name)
 }
 
 gchar *
-fu_uefi_guess_esp_path (void)
+fu_uefi_validate_esp_path (const gchar *extra_path)
```

Comment:
I think if we're validating, we should be returning a GError on failure.

Reply:
Sounds good, I'll adjust.

Reply:
Adjusted